### ISSUE TITLE
Remove default keybindings

### ISSUE THREAD

**Issue by Steellow** [fuhsjr00/bug.n#187]
I added some of my own keybindings to config.ini and they work wonders. I'm just wondering if there's any way of deleting the default ones? They seem to remain even after adding your own

win+space is used to toggle windows taskbar, but it's also the windows shortcut of changing keyboard layout. I can't change keyboard layout with any shortcut now that it's bound to showing/hiding taskbar

Thanks!

**Comment by joten** [comment]
You may delete a bug.n hotkey, by setting it without a value in Config.ini, e.g. `Config_hotkey=#Space::`; have a look at [doc/Customization](https://github.com/fuhsjr00/bug.n/blob/master/doc/Customization.md) for more information.
Deleting Windows' hotkeys is mostly out of scope of bug.n, only <kbd>Win</kbd><kbd>L</kbd> has been considered.

**Comment by Steellow** [comment]
Yes I've set custom hotkey in config.ini which works, but I cant seem to be able to delete the default hotkey for toggling windows taskbar.

I have set this in config.ini: `Config_hotkey=#^Space::Monitor_toggleTaskBar()`

Se when I don't have bug.n open I can change keyboard layout with win+space

When I have bug.n open win+space just toggles the windows taskbar, even though I have changed it's shortcut on config.ini Both win+space and ctr+win+space toggles the windows taskbar.

**Comment by joten** [comment]
Did you unset the default hotkey, too? I.e.:
```
Config_hotkey=#^Space::Monitor_toggleTaskBar()
Config_hotkey=#Space::
```
